image-layout.md

```diff
@@ -134,7 +134,7 @@ The `imageLayoutVersion` value will align with the OCI Image Specification versi
 
 ### oci-layout Example
 
-```json
+```json,title=OCI%20Layout&mediatype=oci%2Dlayout
```

Comment:
@stevvooe 
Sorry for late response. This is because of https://github.com/opencontainers/image-spec/blob/master/schema/spec_test.go#L70. It means that only those who have `mediatype` JSON key will be validated. Any arbitrary string here will not influence it to be joined to spec-test function. I use `oci-layout`, as to close to description in `image-layout-schema.json`.
If we need a nicer name here?